### DISCUSSION TITLE
Invalidating cached pages

### DISCUSSION THREAD

**Discussion by stormfar** [vercel/next.js#10945] [Help]
The new SSG feature is awesome, especially with the `fallback: true` property for dynamic path pages. But I have a question: once a page has been built and cached, how do you invalidate it without rebuilding the app?

**Comment by yjisme** [comment]
same question

**Comment by janus-reith** [comment]
Related: https://github.com/zeit/next.js/discussions/10721#discussioncomment-683

**Reply by lstellway** [reply]
Not sure if I misunderstand the question, but I'm not sure that is what they were after? 

There's another discussion [here](https://github.com/vercel/next.js/discussions/22384) on manually purging the cache. I'm currently trying to test this strategy on my own.
